docs/client/AccountInvite.md

```diff
@@ -1,15 +1,16 @@
 # AccountInvite
 
 ## Properties
+
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**Account** | [***Link**](Link.md) |  | 
+**Account** | [**Link**](Link.md) |  | 
```

Comment:
I've noticed this removal of one "*" from a number of Links. Is this a fix also?